### PR TITLE
Announce opening nominations

### PR DESCRIPTION
**Pull request by patricia-gallardo**
Create a post announcing that the nominations are open and adding Tiril as an author

### CODE REVIEW

File: _pages/author-tiril.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-md-8">
+      <div class="row align-items-center mb-5">
+        <div class="col-md-9">
+          <h2 class="font-weight-bold">
+            {{page.title}}
+            <span class="small btn btn-outline-success btn-sm btn-round"
+              ><a href="{{ site.authors.tiril.twitter }}">Follow</a></span
```
Comment (coderbyheart) [inline _pages/author-tiril.html]: Use `linkedin` property from profile entry:

```suggestion
              ><a href="{{ site.authors.tiril.linkedin }}">Follow</a></span
```

Comment (patricia-gallardo) [reply _pages/author-tiril.html]: I did that but the icon was the twitter bird? So I reverted it again

Comment (coderbyheart) [reply _pages/author-tiril.html]: Ah, it's the hardcoded icon handling here, which is the problem: https://github.com/tech-women/tech-women.github.io/blob/3c3a5f6934ca5c2c3eb3cee30a56f09f3d3be0ae/_pages/authors-list.html#L34

**Comment by coderbyheart** [comment]
@patricia-gallardo I added 7caff106b3f7ec89e05fcb0510c8bcfaabc02b44 which fixes the link icons on the authors page:

Screenshot_20200308_212124

**Comment by patricia-gallardo** [comment]
That's awesome! The "Follow" button on the post doesn't link to anything though since it only links to Twitter

**Comment by coderbyheart** [comment]
Made the template check which type of link is, and if it exists at all: 3643dfc893b8eff6e05cc85fbcf2423c3dec033b

**Pull request merged**